.github/workflows/build-wheels.yml

```diff
@@ -62,48 +62,84 @@ jobs:
           GH_TOKEN: ${{ github.token }}
 
   build_wheels:
-    name: Build wheels on ${{ matrix.os }}
+    name: Wheel ${{ matrix.buildplat[0] }}-${{ matrix.buildplat[1] }}-${{ matrix.python }}
     # TODO(paris): Add this back once generate_backwards_compatibility_data is confirmed to work.
     # needs: generate_backwards_compatibility_data
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.buildplat[0] }}
     strategy:
       matrix:
-        os: [ubuntu-22.04, macos-12, windows-2022]
-        # `windows-2022, ` blocked by https://github.com/pybind/pybind11/issues/3445#issuecomment-1525500927
```

Comment:
Please add a comment about why no windows, x-ref a shortcut story for follow-up.

Reply:
Done